code/modules/mafia/controller.dm

```diff
@@ -704,7 +762,7 @@
 	ui = SStgui.try_update_ui(user, src, null)
 	if(!ui)
 		ui = new(user, src, "MafiaPanel")
-		ui.set_autoupdate(FALSE)
+		ui.set_autoupdate(TRUE)
```

Comment:
Remove the whole line instead. And the reason this was here was because the panel has to build a lot of data for every participant, you should move sizeable chunk of it to static data if you want to switch this.

Reply:
Hmm... I should really keep it on second thought.